### DISCUSSION TITLE
Implement Terraform Provider for StackStorm

### DISCUSSION THREAD

**Discussion by daniel-rocha-ciandt** [StackStorm/st2#5357] [Ideas]
I would like to suggest to start a new implementation of the Terraform Provider for StackStorm (The Providers can be found here: 
https://registry.terraform.io/browse/providers), in order to administrate all resources (Actions, Rules, Packs, Triggers) from a 
single repository. 

One good example is the New Relic's Terraform Provider (https://registry.terraform.io/providers/newrelic/newrelic/latest/docs) that allows you to control everything from Alerts to Dashboards as a code. 

Thanks in Advanced.

**Comment by arm4b** [comment]
Great idea, +100
